### ISSUE TITLE
[Feature] process check after download

### ISSUE THREAD

**Issue by shishirong** [ScoopInstaller/Scoop#6122]
## Feature Request
process check after download 
#### Is your feature request related to a problem? Please describe.
I download app must run proxy app in china, but scoop checked proxy app process is running then break download&update when update proxy app.

#### Describe the solution you'd like
app process running check after download.

**Comment by HUMORCE** [comment]
You can download it first, then close the app and perform the update.

```
scoop download <app>
kill -name <process_name>
scoop update <app>
```

**Comment by shishirong** [comment]
yes It is work, I'm not used download sub command before, sorry.
but I still think process check after download is better when scoop update
